src/Cardano/Wallet/DB.hs

```diff
@@ -62,13 +94,18 @@ data DBLayer m s = DBLayer
         :: PrimaryKey WalletId
         -> m (Map (Hash "Tx") (Tx, TxMeta))
         -- ^ Fetch the current transaction history of a known wallet. Returns an
-        -- empty map if the wallet isn't found.
+        -- empty map if the wallet istn't found.
```

Comment:
this was not necessary

Reply:
Erf. Seems like a merge resolution gone wrong.